homeassistant/components/media_player/roku.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, host):
         self.current_app = None
         self._device_info = {}
 
-        """fetch device info right away"""
+        # fetch device info right away
         self.update()
```

Comment:
Remove this and instead call `add_entities(entities, True)`. That will make sure that `update` is called during entity addition.